{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"sendFailed","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"errno"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5Int32V","text":"Int32"},{"text":")","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}]}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/SwiftFTR\/documentation\/SwiftFTR","doc:\/\/SwiftFTR\/documentation\/SwiftFTR\/TracerouteError"]]},"metadata":{"externalID":"s:8SwiftFTR15TracerouteErrorO10sendFailedyACs5Int32V_tcACmF","roleHeading":"Case","symbolKind":"case","title":"TracerouteError.sendFailed(errno:)","modules":[{"name":"SwiftFTR"}],"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"sendFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"errno","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5Int32V","text":"Int32","kind":"typeIdentifier"},{"text":")","kind":"text"}]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftftr\/tracerouteerror\/sendfailed(errno:)"]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftFTR\/documentation\/SwiftFTR\/TracerouteError\/sendFailed(errno:)"},"sections":[],"abstract":[{"type":"text","text":"Sending a probe failed; the associated errno is provided."}],"references":{"doc://SwiftFTR/documentation/SwiftFTR":{"role":"collection","kind":"symbol","identifier":"doc:\/\/SwiftFTR\/documentation\/SwiftFTR","title":"SwiftFTR","abstract":[{"text":"Massively parallel, async\/await traceroute for macOS using ICMP datagram sockets (no sudo required).","type":"text"}],"url":"\/documentation\/swiftftr","type":"topic"},"doc://SwiftFTR/documentation/SwiftFTR/TracerouteError/sendFailed(errno:)":{"role":"symbol","title":"TracerouteError.sendFailed(errno:)","type":"topic","kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"sendFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"errno"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5Int32V","text":"Int32"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"Sending a probe failed; the associated errno is provided."}],"identifier":"doc:\/\/SwiftFTR\/documentation\/SwiftFTR\/TracerouteError\/sendFailed(errno:)","url":"\/documentation\/swiftftr\/tracerouteerror\/sendfailed(errno:)"},"doc://SwiftFTR/documentation/SwiftFTR/TracerouteError":{"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"TracerouteError","kind":"identifier"}],"url":"\/documentation\/swiftftr\/tracerouteerror","kind":"symbol","abstract":[{"text":"Errors that can occur while performing a traceroute.","type":"text"}],"type":"topic","title":"TracerouteError","identifier":"doc:\/\/SwiftFTR\/documentation\/SwiftFTR\/TracerouteError","navigatorTitle":[{"text":"TracerouteError","kind":"identifier"}]}}}